vendor/licenses/grammar/Atom-PostScript.txt

```diff
@@ -1,7 +1,7 @@
 ---
 type: grammar
 name: Atom-PostScript
-version: 7f562cdfd751ae1863d6b88621e82cd27511ba8a
+version: 5d33239f382c26e96daa1db7dd42559d824b2485
```

Comment:
Could you do that again?

Nah, I'm kidding. ❤️  Thanks!

Reply:
🤔 that seems to have only updated the license and not the submodule ref.

Reply:
Sorted!